testsuite/tests/kuadrant/identical_hostnames/test_identical_hostnames_auth_on_routes.py

```diff
@@ -28,8 +28,8 @@ def test_identical_hostnames_auth_on_routes_rejected(client, authorization, auth
     Tests that 2nd AuthPolicy is rejected on 'route2' declaring identical hostname as 'route' with another
     AuthPolicy already successfully enforced on 'route'.
     Setup:
-        - Two HTTPRoutes declaring identical hostnames but different paths ('/' and '/anything/')
-        - Empty AuthPolicy enforced on the '/' HTTPRoute
+        - Two HTTPRoutes declaring identical hostnames but different paths ('/nothing/' and '/anything/')
```

Comment:
how using `/get` and `/anything` instead? To limit number of endpoints we require from httpbin

Reply:
I had it like that originally but it turns out that completely different paths are needed. The thing is that `/anything` is "subpath" of `/` so they are not completely different and the scenario is not reproducible. In the example in doc `/foo` and `/bar` paths are used so I used `/anything` and `/nothing`.

Btw, why do we need to limit the number of httpbin endpoints?

Reply:
`/nothing` is also "subpath" of `/`, so I still do not see how `/get` is different

The less endpoints we use on httpbin, the easier is to replace it in the future and our dependency on it is lower. It is not a big deal if we require the functionality but I do not think we do in thius case

Reply:
sorry, bad wording. I try again. I need two httproutes declaring identical hostnames but having different paths configured for backend. By different paths I also mean that one path must not be "subpath" of the other one. So in this PR I moved away from `/` and `/anything/` to `/nothing/` and `/anything/`

Reply:
Ok so it is just a prefix you are not changing what httpbin endpoints you are accessing?

Reply:
Yes, just a prefix defined for backend in each httproute. However, I think I know what you are getting at now. I am mixing together backend prefixes in httproute CR and httpbin endpoints. `/anything/` is not ideal candidate for backend prefix because it is also actual httpbin endpoint (see https://eu.httpbin.org/#/Anything) So I should probably use completely different backend prefixes. Maybe something descriptive like `/route1/` and `/route2/`, wdyt?

Reply:
> Maybe something descriptive like `/route1/` and `/route2/`, wdyt?

This is perfect! Also yes, that was exactly what confused me :D

Reply:
This works slightly different than I thought, the backend (httpbin) must have the endpoint that matches the `pathPrefix`. So I updated the PR to use `/anything/route1` and `/anything/route2`. It makes our test suite dependent on Anything HTTPBin endpoint but I can't see a way not to introduce this dependency

Reply:
`/anything` is fine, we already have that